template/package.json

```diff
@@ -35,29 +35,6 @@
     "preact-router": "^2.5.7"
   },
   "jest": {
-    "verbose": true,
-    "setupFiles": [
-      "<rootDir>/tests/__mocks__/browserMocks.js"
-    ],
-    "testRegex": "(/(__tests__|tests)/.*|(\\.|/)(test|spec))\\.jsx?$",
-    "testPathIgnorePatterns": ["/node_modules/", "<rootDir>/tests/__mocks__/*"],
-    "testURL": "http://localhost:8080",
-    "moduleFileExtensions": [
-      "js",
-      "jsx"
-    ],
-    "moduleDirectories": [
-      "node_modules"
-    ],
-    "moduleNameMapper": {
-      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tests/__mocks__/fileMock.js",
```

Comment:
Wondering if we should add that to the preset or if it's something we expect the user to do. Right now the preset doesn't transform any of those.

Reply:
After some googling i think it's better when we leave this part to the user. There are valid use cases to use transformers for `svg` or other files that we can't forsee.